src/app/utils/helper.ts

```diff
@@ -55,25 +62,25 @@ export function getTicker(name: string) {
 
 export function getTruncatedAddress(address: string) {
   if (address) {
-    return `${address.substring(0, 4)}...${address.substring(
-      address.length - 4,
-      address.length,
-    )}`;
+    return `${address.substring(0, 4)}...${address.substring(address.length - 4, address.length)}`;
   }
 }
 
 export function getShortTruncatedAddress(address: string) {
```

Comment:
This one is a little confusing, shouldn't the 4 character one above be the shorter truncated address?

Reply:
In general I think we need more characters shown for the addresses. 6 characters from beginning and end would be better.